app/views/users/edit.html.erb

```diff
@@ -12,9 +12,9 @@
     <%= f.text_field(:name) %><br>
     <%= f.label('Profile Photo') %><br>
     <%= image_tag(url_for(@user.display_image)) + '<br>'.html_safe if @user.avatar.present? %>
-    <%= f.file_field(:avatar) %><br>
+    <%#= f.file_field(:avatar) %><br>
```

Comment:
Can this erb comment be removed?

Reply:
I figured, I wanted to remember to add it back once the active storage issue is figured out